scripts/patch-realsense-ubuntu-lts.sh

```diff
@@ -93,7 +98,9 @@ else
 	echo -e "\e[32mApplying realsense-powerlinefrequency-fix patch\e[0m"
 	patch -p1 < ../scripts/realsense-powerlinefrequency-control-fix.patch
 fi
-
+if [ "$PLATFORM" = "odroid" ]; then
```

Comment:
Improper comparison

Reply:
A single = is actually correct in bash...!